src/main/java/core/basesyntax/FileWriterImpl.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) throws IOException {
+        Files.write(Paths.get(filePath), data.getBytes());
```

Comment:
It is recommended to use try-with-resources to ensure that any resources are automatically closed after the operation is complete. This can help prevent potential resource leaks.